src/main/java/org/team1540/robot2025/subsystems/grabber/GrabberConstants.java

```diff
@@ -0,0 +1,12 @@
+package org.team1540.robot2025.subsystems.grabber;
+
+public class GrabberConstants {
+    // TODO: Fix this
+    public static final int GRABBER_ID = 13;
+    public static final int BEFORE_BREAK_CANDI_ID = 1;
+    public static final int AFTER_BREAK_CANDI_ID = 2;
+    public static final String CANDI_BUS = "canivore";
+
+    public static final double GRABBER_MOI = 0.025;
+    public static final double GRABBER_GEAR_RATIO = 24.0 / 36.0;
```

Comment:
"`GRABBER_`" doesn't need to be part of the name of these constants, since they're already part of the `GrabberConstants` file so when you access them you'll know what they are. If anything, you could think about prefixing them with "`MOTOR_`" so it's clear what hardware you're referring to.